index.html

```diff
@@ -3,14 +3,113 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name = "viewport" content = "width=device=width, intial=scale=1">
```

Comment:
So, the content here has an incorrect value. It should be: `content="width=device-width, initial-scale=1.0"`

Otherwise it does not scale properly.